addons/analytic_user_function/analytic_user_function.py

```diff
@@ -116,8 +116,8 @@ def on_change_account_id(self, cr, uid, ids, account_id, user_id=False, unit_amo
             res ['value']['general_account_id']= a
         return res
 
-    def on_change_user_id(self, cr, uid, ids, user_id, account_id, unit_amount=0):
-        res = super(hr_analytic_timesheet, self).on_change_user_id(cr, uid, ids, user_id)
+    def on_change_user_id(self, cr, uid, ids, user_id, is_timesheet, context, unit_amount=0, account_id=None):
+        res = super(hr_analytic_timesheet, self).on_change_user_id(cr, uid, ids, user_id, is_timesheet, context=context)
```

Comment:
You call super without giving all arguments from the parent, is it normal ? Normally we don't do that.